app/models/user.rb

```diff
@@ -0,0 +1,22 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
+  before_create :calculate_graduation_year
```

Comment:
do this on before save as on the front end we won't be showing the computed value anymore. this will always be calculated and saved